draft-ietf-tls-ticketrequests.md

```diff
@@ -228,11 +225,13 @@ prioritize sending application data over tickets when possible.
 # Security Considerations
 
 Ticket re-use is a security and privacy concern. Moreover, clients must take care when pooling
-tickets as a means of avoiding or amortizing handshake costs. If servers do not rotate session
+tickets as a means of avoiding or amortizing connection costs. If servers do not rotate session
```

Comment:
This one should maybe stay as 'handshake'

Reply:
I don't feel strongly, so I'll swap it back.